src/internal/connector/onedrive/item.go

```diff
@@ -191,12 +191,24 @@ func oneDriveItemMetaInfo(
 
 	perm, err := service.Client().DrivesById(driveID).ItemsById(*itemID).Permissions().Get(ctx, nil)
 	if err != nil {
-		return Metadata{}, errors.Wrapf(err, "failed to get item permissions %s", *itemID)
+		return Metadata{}, err
 	}
 
+	uperms := filterUserPermissions(perm.GetValue())
+
+	return Metadata{Permissions: uperms}, nil
+}
+
+func filterUserPermissions(perms []models.Permissionable) []UserPermission {
 	up := []UserPermission{}
 
-	for _, p := range perm.GetValue() {
+	for _, p := range perms {
+		if p.GetGrantedToV2() == nil {
+			// For link shares, we get permissions without a user
+			// specified
+			continue
```

Comment:
since we're skipping these please add a note about this as a `known issue` in the changelog. We'll probably also have to do a website doc update for it at some point

Reply:
For links shares, IIUC while we can recreate link shares, I don't think we can keep the same URLs. So I'm not sure if there is a point in restoring link shares, at least as of now.